sap/cli/startrfc.py

```diff
@@ -144,6 +169,10 @@ def install_parser(self, arg_parser):
 
         arg_parser.add_argument('-o', '--output', choices=FORMATTERS.keys(), default=next(iter(FORMATTERS.keys())),
                                 help='Output format')
+        arg_parser.add_argument('-R', '--response-file', type=str, default=None,
+                                help=('Dump the entire response to the given file ',
+                                      'which must not exist. The contets format '
```

Comment:
I used to use contents because but I think content is more appropriate:
https://english.stackexchange.com/questions/56831/file-content-vs-file-contents